scripts/bookmarklet.tmpl.js

```diff
@@ -0,0 +1,54 @@
+/* global jQuery, window, document */
+(function(window, document) {
+  var toast = document.createElement('div');
+  var b = document.getElementsByTagName('body')[0];
+  var version = null;
+
+  setStyles(toast)([
+    ['position' , 'fixed'],
+    ['top' , '20px'],
+    ['right' , '30%'],
+    ['padding' , '5px 10px'],
+    ['zIndex' , 1001],
+    ['fontSize' , '18px'],
+    ['color' , '#222'],
+    ['backgroundColor' , '#DFF2BF'],
+    ['text-align', 'center'],
+    ['border', 'solid 2px black'],
+    ['border-radius', '25px']
+  ]);
```

Comment:
> I made it an array just for simpler iteration.

Don't forget `for ... in` loops. :)

```javascript
for (var prop in styles) {
  el.style[prop] = styles[prop];
}
```

As mentioned in my previous comment, though, I don't see what advantage `setStyles` provides over assigning the style properties directly in this case.